Readability.js

```diff
@@ -844,18 +844,50 @@ Readability.prototype = {
 
         this._initializeNode(topCandidate);
       } else if (topCandidate) {
+        parentOfTopCandidate = topCandidate.parentNode;
+
+        // Find a better top candidate node if it contains (at least three) nodes which belong to `topCandidates` array
+        // and whose scores are quite closed with current `topCandidate` node.
+        var alternativeCandidateAncestors = [];
+        for (var i = 1; i < topCandidates.length; i++) {
+          if (topCandidates[i].readability.contentScore / topCandidate.readability.contentScore >= 0.75) {
+            alternativeCandidateAncestors.push(this._getNodeAncestors(topCandidates[i]));
+          }
+        }
```

Comment:
I think it's clear enough, but I'm aware that because we can't assume `let` exists (at least, historically we haven't been able to - maybe it's better now? Does `node` 6.0 support `let`, and does our iOS runtime environment? Esp. for the last question, I don't know the answer...

In any case, it works like this so we can move the refactoring and/or use of `let` to a separate issue.